hubblestack/extmods/returners/sumo_nebula_return.py

```diff
@@ -56,27 +57,27 @@ def returner(ret):
             return
         else:
             for query in data:
-                for key, value in query.items():
-                    if key == 'query_result':
-                        for d in value['data']:
-                            event = {}
-                            event.update({'query': query['query_name']})
-                            event.update({'job_id': jid})
-                            event.update({'minion_id': minion_id})
-                            event.update({'dest_host': fqdn})
-                            event.update({'dest_ip': fqdn_ip4})
-                            event.update(cloud_details)
-                            event.update(d)
-                            event_time = d.get('time', '')
-                            try:
-                                if (datetime.fromtimestamp(time.time()) - datetime.fromtimestamp(
-                                        float(event_time))).days > 365:
-                                    event_time = ''
-                            except:
-                                event_time = ''
-                            finally:
-                                rdy = json.dumps(event)
-                                requests.post('{}/'.format(sumo_nebula_return), rdy)
+                for query_name, query_results in query.iteritems():
+                    if 'data' not in query_results:
+                        query_results['data'] = [{'error': 'result missing'}]
+                    for query_result in query_results['data']:
+                        event = {}
+                        event.update(query_result)
+                        event.update({'query': query_name})
+                        event.update({'job_id': jid})
+                        event.update({'minion_id': minion_id})
+                        event.update({'dest_host': fqdn})
+                        event.update({'dest_ip': fqdn_ip4})
+                        event.update({'dest_fqdn': local_fqdn})
+                        event.update({'system_uuid': __grains__.get('system_uuid')})
+
+                        event.update(cloud_details)
+                        try:
+                            rdy = json.dumps(event)
+                            requests.post('{}/'.format(sumo_nebula_return), rdy)
+                        except:
+                            print('Hit an exception trying to send to sumo! continuing')
```

Comment:
You should use logger instead of 'print' statements. You can take reference of logging from splunk_nova_return or splunk_nebula_return.

Reply:
Agreed. A really minor change and we can get this merged.

Reply:
incoming commit